include/irr/asset/EColorSpace.h

```diff
@@ -5,6 +5,7 @@ namespace irr
 {
 	namespace asset
 	{
+        //! Specifies a color space of an image
```

Comment:
I don't like the vulkan enum split between `NONLINEAR` and `LINEAR`, you should delete these sufficies (and the `KHR` and `EXT`) 

whether the pixels are linear or not will be given by `ELECTRO_OPTICAL_TRANSFER_FUNCTION`

(together the pair of `E_COLOR_SPACE` and `ELECTRO_OPTICAL_TRANSFER_FUNCTION` will give the equivalent Vulkan enum)